<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainToolBar</class>
 <widget class="QToolBar" name="mainToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>38</height>
   </rect>
  </property>
  <action name="action_load_mesh">
   <property name="icon">
    <iconset>
     <normaloff>icons/radar_plot.svg</normaloff>icons/radar_plot.svg</iconset>
   </property>
   <property name="text">
    <string>Load &amp;mesh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset>
     <normaloff>icons/cancel.svg</normaloff>icons/cancel.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_show_tree">
   <property name="icon">
    <iconset>
     <normaloff>icons/list.svg</normaloff>icons/list.svg</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Tree</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_help">
   <property name="icon">
    <iconset>
     <normaloff>icons/document.svg</normaloff>icons/document.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset>
     <normaloff>icons/about.svg</normaloff>icons/about.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>?</string>
   </property>
  </action>
  <action name="action_normal_mode">
   <property name="icon">
    <iconset>
     <normaloff>icons/rotate_camera.svg</normaloff>icons/rotate_camera.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Normal mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_load_block_model">
   <property name="icon">
    <iconset>
     <normaloff>icons/data_sheet.svg</normaloff>icons/data_sheet.svg</iconset>
   </property>
   <property name="text">
    <string>Load &amp;block model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_selection_mode">
   <property name="icon">
    <iconset>
     <normaloff>icons/cursor.svg</normaloff>icons/cursor.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Selection mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_camera_properties">
   <property name="icon">
    <iconset>
     <normaloff>icons/compact_camera.svg</normaloff>icons/compact_camera.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Camera properties</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_load_points">
   <property name="icon">
    <iconset>
     <normaloff>icons/scatter_plot.svg</normaloff>icons/scatter_plot.svg</iconset>
   </property>
   <property name="text">
    <string>Load &amp;points</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_plan_view">
   <property name="icon">
    <iconset>
     <normaloff>icons/upload.svg</normaloff>icons/upload.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Plan view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="action_north_view">
   <property name="icon">
    <iconset>
     <normaloff>icons/up.svg</normaloff>icons/up.svg</iconset>
   </property>
   <property name="text">
    <string>N&amp;orth view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="action_east_view">
   <property name="icon">
    <iconset>
     <normaloff>icons/right.svg</normaloff>icons/right.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;East view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="action_take_screenshot">
   <property name="icon">
    <iconset>
     <normaloff>icons/webcam.svg</normaloff>icons/webcam.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Take screenshot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <addaction name="action_show_tree"/>
  <addaction name="separator"/>
  <addaction name="action_plan_view"/>
  <addaction name="action_north_view"/>
  <addaction name="action_east_view"/>
  <addaction name="separator"/>
  <addaction name="action_take_screenshot"/>
 </widget>
 <resources/>
 <connections/>
</ui>
